dumper/es_utils.py

```diff
@@ -21,5 +24,16 @@ def iter_data(es_client, index, query):
             yield doc
 
 
+def query_users_data(es_client, index, query):
```

Comment:
@snopoke You should confirm that this is the right way to do this. I adapted from above, but couldn't test.